terratest/test/helpers.go

```diff
@@ -36,6 +39,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var dnsZone = getEnv("GSLB_DOMAIN", "cloud.example.com")
+var dnsServer1 = getEnv("DNS_SERVER1", "localhost")
+var dnsServer1Port = getEnv("DNS_SERVER1_PORT", "5053")
+var dnsServer2 = getEnv("DNS_SERVER2", "localhost")
+var dnsServer2Port = getEnv("DNS_SERVER2_PORT", "5054")
+var primaryGeoTag = getEnv("PRIMARY_GEO_TAG", "eu")
+
```

Comment:
consider create file `init.go` and use this instead:

```go
package test

import "github.com/AbsaOSS/gopkg/env"

type clusterSettings struct {
	DNSZone       string
	PrimaryGeoTag string
	DNSServer1    string
	Port1         string
	DNSServer2    string
	Port2         string
}

var (
	settings clusterSettings
)

func init(){
	settings = clusterSettings{
		DNSZone:       env.GetEnvAsStringOrFallback("GSLB_DOMAIN", "cloud.example.com"),
		PrimaryGeoTag: env.GetEnvAsStringOrFallback("PRIMARY_GEO_TAG", "eu"),
		DNSServer1:    env.GetEnvAsStringOrFallback("DNS_SERVER1","localhost"),
		Port1:         env.GetEnvAsStringOrFallback("DNS_SERVER1_PORT","5053"),
		DNSServer2:    env.GetEnvAsStringOrFallback("DNS_SERVER2","localhost"),
		Port2:         env.GetEnvAsStringOrFallback("DNS_SERVER2_PORT","5054"),
	}
	// add different things needs to be initialised here....
}

```
`init()` is used for initializing packages, function should start before tests (btw: after variable initialison)